tests/app/celery/test_broadcast_message_tasks.py

```diff
@@ -137,6 +137,25 @@ def test_send_broadcast_event_creates_zendesk_p1(mocker, notify_api, sample_broa
     assert "Dear Sir/Madam" in zendesk_args['message']
 
 
+def test_send_broadcast_event_doesnt_p1_when_cancelling(mocker, notify_api, sample_broadcast_service):
+    template = create_template(sample_broadcast_service, BROADCAST_TYPE)
+    broadcast_message = create_broadcast_message(
+        template,
+        status=BroadcastStatusType.BROADCASTING,
+        areas={'areas': ['wd20-S13002775', 'wd20-S13002773'], 'simple_polygons': []},
+    )
+    create_broadcast_event(broadcast_message, message_type=BroadcastEventMessageType.ALERT)
```

Comment:
Do we need this one if we're just testing the cancellation?

Reply:
i wasn't sure, but it's more correct since you need an alert to have a cancel, and was worried that there might be data integrity concerns that might lead to unexpected behaviour.